news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "\"Medal Hunter\" Title's Second Coming"
+date: 2023-08-28 03:00:00 +0000
+---
+
+The legendary "**Medal Hunter**" title came back once again challenging every player to find a solution for a new medal!
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/banner.jpg)
+
+[Medals](/wiki/Medals) are osu! achievements which are given for achieving certain goals, and the last batch of medals was quite exciting.
+On 2023-07-27, alongside new [Hush-Hush medals](/wiki/Medals#hush-hush), [Ephemeral](https://osu.ppy.sh/users/102335) made a [news post](https://osu.ppy.sh/home/news/2023-07-27-summer-splash) in which he announced the "**Medal Hunter**" title coming back once again.
+
+"**Medal Hunter**" is a [unique user title](/wiki/People/Users_with_unique_titles), which is given for being the first player to unlock an extremely hard to solve Hush-Hush medal. The [previous "**Medal Hunter**" title](https://osu.ppy.sh/home/news/2022-09-09-how-zetera-became-a-medal-hunter) has been given out nearly **6 years ago**, which definitely made it even more interesting.
+
+In order to obtain such a desirable title, you had to be the first player to find the solution for the **Internment** medal and unlock it. **Internment** is a Hush-Hush medal added in the last batch, being so hint-based that developers found unlocking it worth a title. And while only one player could get a shiny "**Medal Hunter**" text under their username, many still have decided to team up with others on the [Medal Hunters Discord server](https://discord.gg/8qpNTs6)!
+
+The community was trying their best to *hunt* for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. *On this journey of basking in the shade of an ailing mind*, the community had received various hints, including a video recorded by one of medal hunters at [cavoe's osu! event (COE)](/wiki/Community/cavoe's_osu!_event), featuring [peppy](https://osu.ppy.sh/users/2) himself!
```

Comment:
I would suggest adding the video as an embed in the news post to break the huge blocks of text and for everyone to be able to view the video. Outside of referencing it here there is no link or mention to where one would see this video

also wala already suggested this